spk/lua/src/service-setup.sh

```diff
@@ -0,0 +1,16 @@
+service_postinst ()
+{
+	echo "Creating startup scripts for Lua executables" >> ${INST_LOG}
+	for executable in ${SYNOPKG_PKGDEST}/bin/lua*; do
+		ln -s "$executable" /usr/local/bin/$(basename "$executable")
+	done
```

Comment:
@hgy59 Notice: I find it interesting... such logic may move to framework to reduce code in package, up to a point "service-setup.sh" may be removed for command line tools

Reply:
Thanks. I created it this way so that the service-setup.sh file doesn't need to be updated when a version update of the package is published.

Reply:
I am not a fan of * here, because you can not see the affected names.

For the framework solution I suppose we can use the PLIST to get the names of the target files in bin and sbin folders (and maybe lib folder?) to create a "service-setup.sh" file that creates and removes links. 
Maybe we call the generated file "cli-setup.sh" and activate it with `MAKE_CLI_SETUP = 1`.
If this is based on the PLIST we get the right version anyhow.